src/components/Common/Button/index.tsx

```diff
@@ -69,13 +86,29 @@ const Button: React.FC<ButtonProps> = ({
       buttonStyle.push('px-4 py-2 text-sm');
   }
   if (className) {
-    buttonStyle.push(className);
+    buttonStyle.push(className ?? '');
   }
```

Comment:
Curious. Why do we need nullish coalescing here if we are checking if className is set above?

Reply:
Typescript didnt seem to understand it was a string

Reply:
Our hard-earned A+ LGTM rating though 😭

Reply:
Just remove the wrapping conditional then?

Reply:
Just remove the wrapping conditional then?